src/App.css

```diff
@@ -0,0 +1,4 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] you already have this style in your index.css, kindly remove here to make your code base cleaner